src/main/java/seedu/address/model/person/Avatar.java

```diff
@@ -0,0 +1,45 @@
+package seedu.address.model.person;
+
+import java.io.FileInputStream;
+import java.io.FileNotFoundException;
+
+import javafx.scene.image.Image;
+
+/**
+ * Represents displayed photo of a contact in the address book.
+ */
+public class Avatar {
+
+    private static final String DEFAULT_PATH = "/images/default_photo.png";
+    private String path;
+
+    public Avatar() {
+        this.path = DEFAULT_PATH;
+    }
+
+    public Avatar(String path) {
+        this.path = path;
+    }
+
+    public Avatar(Avatar avatar) {
+        this.path = avatar.getPath();
+    }
+
+    public void setAvatar(String path) {
+        this.path = path;
+    }
+
+    public Image getImage() {
+        try {
+            return new Image(new FileInputStream(path));
+        } catch (FileNotFoundException e) {
```

Comment:
Consider the event when the path given by the user is invalid. This results in the interface still showing a successful execution of command.

Perhaps consider having an `Image` attribute, with its default value being `new Image(this.getClass().getResourceAsStream(DEFAULT_PATH)` as per your code. So your `getImage()` method returns the `Image` itself